src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/WebhookIssueController.cs

```diff
@@ -31,15 +32,17 @@ public IActionResult Index()
             // process has been started > in logger and time
         }
 
-        [HttpGet("{id}")]
-        public async Task<ActionResult> GetProduct(int id)
+        [HttpGet("{owner}/{repo}/{id}")]
+        [HttpGet("/{owner}/{repo}/{id}")]
```

Comment:
What is the reason for having both routes here?

Reply:
Ah good you found this. I had this in place to ask if you have a preference on either one?

Reply:
I believe the difference is that if the _controller_ also has a route then using `foo/bar` will be _appended_ to that. If you use `/foo/bar` then it will always be at the root.

Reply:
Here's the docs: https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-3.1#combining-attribute-routes

Reply:
In other words, this action will handle requests to:

`/api/WebhookIssue/{owner}/{repo}/{id}`

And also:

`/{owner}/{repo}/{id}`

Reply:
OK thanks. Removed unused API route.